### ISSUE TITLE
What happens when I define both time_limit and distance_limit in our Isochrone API?

### ISSUE THREAD

**Issue by oblonski** [graphhopper/directions-api-doc#24]
For example, if time_limit is unspecified, the default is 600. What happens when I now specify distance_limit = 50000?

**Comment by oblonski** [comment]
Do I need to increase time_limit accordingly?

**Comment by oblonski** [comment]
I would expect that if I specify the distance_limit, the time_limit is ignored entirely, i.e. it only makes sense to specify either time_limit OR distance_limit.

**Comment by karussell** [comment]
Yes, only one makes sense. And [if distance_limit > 0 we ignore time_limit](https://github.com/graphhopper/graphhopper/blob/master/web-bundle/src/main/java/com/graphhopper/resources/IsochroneResource.java#L89)

**Comment by oblonski** [comment]
Ok. Super. Then we need to clarify it in the docs. What is the max. distance_limit? We only have values for time_limits in our pricing schema.

**Comment by karussell** [comment]
It is derived from the time_limits but we likely need to rethink this part together.
